manual/usage/backup_and_restore.md

```diff
@@ -0,0 +1,62 @@
+## Overview
+
+There are generally two parts of data to backup
+- SeaSearch index metadata
+- SeaSearch index document data
+
+## Backuping up and restore seaseach data
+For different backend storage type, there are some differences in the data that needs to be backed up.
+
+There are two backup processes, depending on the configured storage backend:
+- backup full data (For local disk storage).
+- only backup backup the metadata (For object storage e.g. s3).
```

Comment:
There are two backup strategies, depending on the configured storage backend:
- If you use local storage, you need to backup both index metadata and document data
- If you use object storage, you only need to backup index metadata. Whether and how the document data needs backup depends on your object storage setup.